### PR TITLE
added Rat-a-Tat Cat game

### PR DESCRIPTION
**Pull request by omriattiya**
Submitting this PR means you have:

- [x] searched the [readme](https://github.com/edm00se/awesome-board-games/blob/main/readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/main/formatting.md)
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines](https://github.com/edm00se/awesome-board-games/blob/main/contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

This PR is for Hacktoberfest #176

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -833,6 +833,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | --------:|
 |   2 - 4 |      8+  |      15m |
 
+### [Rat-a-Tat Cat](https://www.boardgamegeek.com/boardgame/3837/rat-tat-cat)
+
+> Get rid of the rats and go for the cats! In Rat-a-Tat Cat, less is always better, and you want to go out with the lowest score. Can you remember the numbers on the other players' cards? Can you keep a poker face, but notice when another player looks pleased? Sharpen your memory and your timing, and have fun with the cool cats and bad rats of Rat-a-Tat Cat. This is the Gamewright version.
+
+![Rat-a-Tat Cat image](https://cf.geekdo-images.com/4nQQ9GA-1JXcxwNd-tCR8w__imagepage/img/5-81C1cMjLUJ36rrR6YuxMJILHs=/fit-in/900x600/filters:no_upscale():strip_icc()/pic147435.jpg)
```
Comment (edm00se) [inline readme.md:840]: The image is oversized for what the formatting guide targets. I found an alternative source that better first the target size.

```suggestion
![Rat-a-Tat Cat image](https://cf.geekdo-images.com/4nQQ9GA-1JXcxwNd-tCR8w__itemrep/img/7bCQhZauNwAuekRT3xwbt2FCLlo=/fit-in/246x300/filters:strip_icc()/pic147435.jpg)
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Thank you for contributing @omriattiya. The image size is a bit big. I've suggested an alternate source that should be a better match. Once that's updated I'll be happy to approve and merge this PR.

**Comment by omriattiya** [comment]
@edm00se updated

**Review by edm00se — APPROVED** [review]
Thank you!

**Pull request merged**